src/mon/MgrStatMonitor.cc

```diff
@@ -136,6 +178,7 @@ version_t MgrStatMonitor::get_trim_to()
 
 void MgrStatMonitor::on_active()
 {
+  update_logger();
```

Comment:
Oh, I think this shoudl be guarded by the osdmap.require_osd_release >= CEPH_RELEASE_LUMINOUS so that we don't step on the PGMonitor stats during upgrade